scalableminds/webknossos
# run sbt in docker-compose as non-root user

### Steps to test:
Maybe you have to
```
chown -R "$USER:$USER" ~/.m2 ~/.ivy2 ~/.sbt ~/.yarn-cache
```
Then, run

```
docker pull scalableminds/sbt:sbt-0.13.15_mongo-3.2.17_node-8.x_jdk-8
chown -R "$USER:$USER" .
docker-compose run --rm sbt clean compile
```
Now all generated files should belong to yourself. Future recompilations might be faster (seems to work for me). @fm3, can you confirm this?

------
